convert2rhel/utils.py

```diff
@@ -618,4 +625,44 @@ def set_locale():
     os.environ.update({"LC_ALL": "C", "LANG": "C"})
 
 
+def get_secrets_from_conf_file(path_to_cfg=None):
+    """Get the secret (activation key or password) from config file.
+
+    The order is:
+    1. if filename is set then load from it
+    2. ~/.convert2rhel.ini
+    3. /etc/convert2rhel.ini
+
+    Return {"password": <value>, "activation-key": <value>}.
+    If the secret isn't set, default value is None.
+    """
+    header = "SECRET"  # header in config file
+    secret_names = ("password", "activation-key")
```

Comment:
We can remove that 2 variables here if we opt-in to not use the loop there and go with a simpler version.